python/jwt/security/jwt-hardcode.py

```diff
@@ -1,25 +1,27 @@
 import jwt
 
-# ruleid: jwt-python-hardcoded-secret
 secret_const = "this-is-secret"
 
+
 def bad1():
     # ruleid: jwt-python-hardcoded-secret
-    encoded = jwt.encode({'some': 'payload'}, 'secret', algorithm='HS256')
+    encoded = jwt.encode({"some": "payload"}, "secret", algorithm="HS256")
     return encoded
 
+
 def bad2():
     # ruleid: jwt-python-hardcoded-secret
-    encoded = jwt.encode({'some': 'payload'}, secret_const, algorithm='HS256')
+    encoded = jwt.encode({"some": "payload"}, secret_const, algorithm="HS256")
```

Comment:
Hmmm. Single quoted strings are allowed in python, no? Should the rule capture both cases?

Reply:
Added an additional test case in cbdad21b350a0cbe56155a93d13a7055e6072970